daml/Tests/Trading/TradingIntegrationTest.daml

```diff
@@ -1,190 +1,190 @@
 module Tests.Trading.TradingIntegrationTest where
```

Comment:
Again, this business of commenting out the whole file. Is there some way we can remove it?

Reply:
I have taken a look at what we should do with this file. We need to agree on if we want to do integration testing via a script like this or do we want to preform it manually once the UI is up and running. Since the integration is quite difficult to setup and run on a local machine due to timing (ie, the script can only be ran after the Exberry adapter is up and running as it only picks up new events but the adapter can only be started after the Integration templates have been loaded into the sandbox, etc.) it may be better to find solution that uses the UAT environment instead. 

If we decide to keep the script, the only options we have is to move it outside of the `daml/` directory or to put it into its own folder with its own `daml.yaml` file but the file always show as having an error as it depends on `Setup:Setup` being ran before (this needs to be ran before as the Exberry Adapter and Integration depend on certain parties being created and available in the sandbox in order to be able to start them).

TL;DR - its complicated...